neo.UnitTests/IO/UT_ByteArrayComparer.cs

```diff
@@ -0,0 +1,27 @@
+﻿using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Neo.IO;
+
+namespace Neo.UnitTests.IO
+{
+    [TestClass]
+    public class UT_ByteArrayComparer
```

Comment:
Which calls usually uses this `ByteArrayComparer`?

Reply:
It is used by `Find()` in `DataCache.cs` to compare byte arrays either from cache or storage to provide a sequential return value.